pkg/reconciler/pipelinerun/resources/pipelineref_test.go

```diff
@@ -313,6 +316,24 @@ func TestGetPipelineFunc_RemoteResolution(t *testing.T) {
 			pipelineYAMLStringWithoutDefaults,
 		}, "\n"),
 		wantPipeline: parse.MustParseV1Pipeline(t, pipelineYAMLStringWithoutDefaults),
+	}, {
+		name: "v1 remote pipeline with different namespace",
```

Comment:
I think the test looks a bit confusing with the `wantPipeline` and `ignoreNamespace`. It looks like we are dealing with 2 `pipelines` in different namespaces, but we are dealing is different namespaces between the `pipelienrun` (hidden in the test case with namespace `default`) and `pipeline`.

Maybe it is clearer to rename the test to `v1 remote pipeline in different namespace from pipelinerun`. And instead of comparing against `pipelineYAMLStringWithoutDefaults` with `ignoreNamespace`, I feel it is clearer to just compare the result against the original input, which means the resolution is successful.

Reply:
I agree with renaming the test to make it more clear. I am not sure I understand the other part. I guess I should have a new `pipelineYAMLStringWithNamespace` or something and use it both for `pipelineYAML` and `wantPipeline`, would that work ?

Reply:
Yes, yes, that is what I meant 😀

Reply:
I'm also a bit confused about `ignoreNamespace`, because I think if we removed the logic change in pipelineref.go, this test would still pass. Is that right?

Reply:
Updated